### PR TITLE
Add colors for NERDTree plugin

### PR DESCRIPTION
**Pull request by sbrki**
Hi!

I added colors for the NERDTree plugin, trying to keep it consistent with netrw colors and VSCode theme.

### CODE REVIEW

**Comment by MindTooth** [comment]
Hi,

Thanks for submitting a PR. 👍🏻 

Have you set the colors of the shell as well to match Srcery?  Below is my netrw that has different colors.  I use kitty with the colors from https://github.com/srcery-colors/srcery-terminal.

Screen Shot 2020-12-06 at 12 43 32

Which in terms different from the screenshot in the README. 🤔  I need to debug more on my part.

Screen Shot 2020-12-06 at 12 50 19

@roosta Thought?

**Comment by sbrki** [comment]
>Have you set the colors of the shell as well to match Srcery?

Nope, the screenshot was on some default tango scheme that shipped with my term, however I'm running with true color
enabled, so there shouldn't be any differences.

Which part(s) of the screenshot looks off to you?

**Comment by MindTooth** [comment]
> > Have you set the colors of the shell as well to match Srcery?
>
> Which part(s) of the screenshot look off to you?

Both yours and mine.  The last is correct if I'm not mistaken.

I had a friend test it too, and he ended up with the last one (green).  He has set both terminal and Vim to use the scheme.

I'm unfortunately not that used to using a project drawer in Vim.  So I might not be the right person to conclude either way.

If nothing else this is a good discussion to have and maybe conclude on a scheme that makes Srcery shine more. 😄

**Comment by sbrki** [comment]
>I had a friend test it too, and he ended up with the last one (green). He has set both terminal and Vim to use the scheme.

Hmm, from what I've seen, it looks to me that the last screenshot you sent (the correct one) is not correct :)
For example https://github.com/srcery-colors/srcery-vim/blob/master/colors/srcery.vim#L622 is explicitly set to blue, while in the screenshot it is green.

Also I've tweaked NERDTree colors to match the first screenshot you sent.

**Comment by roosta** [comment]
The screenshot from the readme is outdated, I did a `netrw` overhaul a while back, changing among another things the directory to be blue. (I did notice that if you install NERDTree and run netrw without this pull request the netrw colors are changed, possibly a quirk with NERDTree)

I'm good with a merge on this, and thanks for the contribution!

**Comment by roosta** [comment]
Maybe reword cad72562fb8cde472e02e45e9e2fd23200e1c257 fixing the typo "NERDTRee" to "NERDTree"
